packages/conf-bmake/conf-bmake.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "michipili@gmail.com"
+homepage: "http://www.crufty.net/help/sjg/bmake.htm"
+authors: "sjg@crufty.net"
+bug-reports: "sjg@crufty.net"
+license: "BSD"
+config-state-vars: [
+  path "The detected path a BSD Make compatible make program."
+]
+build: [
+  ["sh" "%{build}%/files/detect_program.sh" "%{prefix}%/lib/conf-bmake" "bmake"] {!(os = "freebsd" | os = "openbsd" | os = "netbsd")}
+  ["sh" "%{build}%/files/detect_program.sh" "%{prefix}%/lib/conf-bmake" "make"] {(os = "freebsd" | os = "openbsd" | os = "netbsd")}
```

Comment:
Good catch! There is something nasty here: if I pin the package to the directory containing the description, as with

~~~~ console
% opam pin add conf-bmake packages/conf-bsd/conf-bsd.1.0
~~~~

then it works with the version I uploaded. If I add my working copy of the repo to opam, then it works with the change you suggested.  I think it should work the same with both approach. Do you see a buggy behaviour there?